redbot/core/commands/requires.py

```diff
@@ -553,10 +553,9 @@ async def _verify_user(self, ctx: "Context") -> bool:
             if user_perms.administrator or user_perms >= self.user_perms:
                 return True
 
-        if self.privilege_level is not None:
-            privilege_level = await PrivilegeLevel.from_ctx(ctx)
-            if privilege_level >= self.privilege_level:
-                return True
+        privilege_level = await PrivilegeLevel.from_ctx(ctx)
+        if privilege_level >= self.privilege_level:
+            return True
```

Comment:
This still needs to check if privilege is not `PrivilegeLevel.NONE` cause we don't want to allow people to use the command if developer used `@bot.has_permissions(...)`